data_structures/src/chain.rs

```diff
@@ -733,6 +787,21 @@ pub struct PublicKey {
     pub bytes: [u8; 32],
 }
 
+/// Public Key data structure
+#[derive(Debug, Default, Eq, PartialEq, Clone, Serialize, Deserialize)]
+pub struct SecretKey {
```

Comment:
Why define a new data structure? Wasn't a `SecretKey` already in use before?

Reply:
Because we want a SecretKey storable, and the other is from Secp256k1